src/components/address.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+export default class Address extends React.Component {
```

Comment:
Let's rewrite `Address` and `Company` to functional components. Functional components are preferable
https://reactjs.org/docs/components-and-props.html#function-and-class-components